sentry-android-fragment/src/main/java/io/sentry/android/fragment/FragmentLifecycleIntegration.kt

```diff
@@ -0,0 +1,74 @@
+package io.sentry.android.fragment
+
+import android.app.Activity
+import android.app.Application
+import android.app.Application.ActivityLifecycleCallbacks
+import android.os.Bundle
+import androidx.fragment.app.FragmentActivity
+import io.sentry.IHub
+import io.sentry.ILogger
+import io.sentry.Integration
+import io.sentry.SentryLevel.DEBUG
+import io.sentry.SentryOptions
+import java.io.Closeable
+
+class FragmentLifecycleIntegration(private val application: Application) :
+    ActivityLifecycleCallbacks,
+    Integration,
+    Closeable {
+
+    private lateinit var hub: IHub
+    private lateinit var logger: ILogger
+
+    override fun register(hub: IHub, options: SentryOptions) {
+        this.hub = hub
+        this.logger = options.logger
+
+        application.registerActivityLifecycleCallbacks(this)
+        logger.log(DEBUG, "FragmentLifecycleIntegration installed.")
+    }
+
+    override fun close() {
+        application.unregisterActivityLifecycleCallbacks(this)
+        logger.log(DEBUG, "FragmentLifecycleIntegration removed.")
```

Comment:
if you don't check if `logger` is `isInitialized`, it'd crash the app when `close` is called, this would be a bug in the SDK, but always better to prevent it, so a simple check would save us some headache.

Reply:
Right, good point. Added check in 87275f641a00eee1dcdec96f1045c538f94f0442